company.rb

```diff
@@ -0,0 +1,65 @@
+$LOAD_PATH << '.'
+require 'quality-assurance'
+require 'human-resource'
+require 'employee'
+require 'module'
+
+
+class Company
+include Read_CSV_File
+  attr_accessor :company_id, :company_name, :company_type, :headquarter, :founded
+  @@no_of_employees = 0
+  
+
+  def initialize(company_id, company_name, company_type, headquarter, founded)
+    @company_id = company_id
+    @company_name = company_name
+    @company_type = company_type
+    @headquarter = headquarter
+    @founded = founded
+    hr_object = HR.new(0o01, 'HR', 'recruitment, selection, placement, socialisation')
```

Comment:
you can move this to a new method and call this method in your initialize otherwise this method will become too big. break it down to smaller units. 

def initialize_departments 
   hr = HR.new(0o01, 'HR', 'recruitment, selection, placement, socialisation')
   qa =QA.new(0o02, 'QA', 'Good')
end